training/a4x/llama3-1-405b/values.yaml

```diff
@@ -0,0 +1,68 @@
+dwsSettings:
+  maxRunDurationSeconds: null
+network:
+  gibVersion: us-docker.pkg.dev/gce-ai-infra/gpudirect-gib/nccl-plugin-gib-arm64:v1.0.7
+  hostNetwork: true
+  ncclSettings:
+  - name: NCCL_DEBUG
+    value: VERSION
+  - name: NCCL_ALGO
+    value: "Ring,Tree"
+  - name: NCCL_NET_GDR_LEVEL
+    value: PIX
+  - name: NCCL_NET_GDR_C2C
+    value: "1"
+  - name: NCCL_P2P_NET_CHUNKSIZE
+    value: "2097152"
+  - name: NCCL_NVLS_ENABLE
+    value: "0"
+  subnetworks[]: null
+queue: null
+targetNodepools: null
+tasSettings:
+  topologyRequest:
+    kueue.x-k8s.io/podset-preferred-topology: kubernetes.io/hostname
```

Comment:
Is this right? Does it need to require SBRG and then prefer a certain number of VMs within a rack?